packages/terra-consumer-nav/src/components/modal/Modal.scss

```diff
@@ -5,13 +5,15 @@
     background: var(--terra-consumer--modal-background, #fff);
     border-radius: 5px;
     left: 20px;
+    max-height: calc(100% - 80px);
```

Comment:
why is this needed?

Reply:
Scroll bar does not appear if max-height is not set. The modal header(title and close button) height is 60px and space below modal content is 10px/20px for mobile/tablet. Subtracted the height of the header and space below content from screen height.

Reply:
I guess I don't understand why a help button fix needs to modify modal styles?

Reply:
Sorry, there was a modal content scrolling issue in Nav. I fixed it as part of the same PR. I can create a separate PR.